composer.json

```diff
@@ -25,8 +26,8 @@
   },
   "repositories": [
     {
-      "type": "composer",
-      "url": "https://code.spryker.com/repo/private"
+      "type": "git",
+      "url": "git@github.com:spryker-eco/adyen-api.git"
```

Comment:
shouldnt be needed if you add it to composer/packagist, right away.

Reply:
Agree, will be deleted after *adyen-api* module is released.